tests/functional/test_contract_instance.py

```diff
@@ -40,6 +40,12 @@ def test_contract_calls(owner, contract_instance):
     assert contract_instance.myNumber() == 2
 
 
+def test_contract_mehtod_calls(owner, contract_instance):
```

Comment:
The name of this strikes the same intent as the test above and encompasses the same. Can they be combined?

Also, `method` is spelled wrong here.